### PR TITLE
Update event descriptions

### PR DESCRIPTION
**Pull request by bafonins**
**Summary:**

Closes https://github.com/TheThingsNetwork/lorawan-stack/issues/540

**Changes:**

- Change event descriptions
- Regenrate messages

**Notes for Reviewers:**

Please check descriptions for consistency.

### CODE REVIEW

File: pkg/applicationserver/observability.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ var (
 	evtReceiveDataDown      = events.Define("as.down.data.receive", "receive downlink data message")
 	evtDropDataDown         = events.Define("as.down.data.drop", "drop downlink data message")
 	evtForwardDataDown      = events.Define("as.down.data.forward", "forward downlink data message")
-	evtLostQueueDataDown    = events.Define("as.down.data.queue.lost", "lost downlink data queue")
+	evtLostQueueDataDown    = events.Define("as.down.data.queue.lost", "lose downlink data queue")
```
Comment (htdvisser) [inline pkg/applicationserver/observability.go]: Not sure about this one. @johanstokking ?

File: pkg/gatewayserver/observability.go [hunk 2]
```diff
@@ -35,8 +35,8 @@ var (
 	evtForwardUp = events.Define("gs.up.forward", "forward uplink message")
 
 	evtSendDown      = events.Define("gs.down.send", "send downlink message")
-	evtTxSuccessDown = events.Define("gs.down.tx.success", "downlink message transmit success")
-	evtTxFailureDown = events.Define("gs.down.tx.fail", "downlink message transmit fail")
+	evtTxSuccessDown = events.Define("gs.down.tx.success", "transmit downlink message success")
```
Comment (htdvisser) [inline pkg/gatewayserver/observability.go]: This sounds more like Yoda than English. Perhaps use `successful`/`failure`?

File: pkg/identityserver/user_registry.go [hunk 3]
```diff
@@ -38,11 +38,11 @@ import (
...
 var (
-	evtCreateUser = events.Define("user.create", "Create user")
-	evtUpdateUser = events.Define("user.update", "Update user")
-	evtDeleteUser = events.Define("user.delete", "Delete user")
+	evtCreateUser = events.Define("user.create", "create user")
+	evtUpdateUser = events.Define("user.update", "update user")
+	evtDeleteUser = events.Define("user.delete", "delete user")
 
-	evtUpdateUserIncorrectPassword = events.Define("user.update.incorrect_password", "Incorrect password for user update")
+	evtUpdateUserIncorrectPassword = events.Define("user.update.incorrect_password", "update user password incorrect")
```
Comment (htdvisser) [inline pkg/identityserver/user_registry.go]: maybe "update user failure: incorrect password"

File: pkg/applicationserver/observability.go (hunk 1 above)
Comment (johanstokking) [reply pkg/applicationserver/observability.go]: Yeah good catch @bafonins. The event names are imperative

**Pull request merged**